.gitignore

```diff
@@ -1,6 +1,7 @@
 ### Bastion BOT settings ###
 settings/credentials.json
 settings/config.json
+data/bastion.db
 data/Bastion.sqlite
```

Comment:
Why is data/Bastion.sqlite being ignored? I only see the bastion.db when bastion starts. no sign of Bastion.sqlite

Reply:
nvm